Assets/Scripts/UI/PickupPup/DogProfile/DogProfileButtonController.cs

```diff
@@ -65,18 +65,20 @@ protected override void setReferences()
 
     #endregion
 
-	public void Init(DogProfile profile, DogDescriptor[] dogs)
+    public void Init(DogProfile profile, DogDescriptor[] dogs)
```

Comment:
I'll look for those later today. The radials work again. Something about removing the actual timer object from the dogCollarSlot items in the editor broke them.